keps/sig-storage/3476-volume-group-snapshot/README.md

```diff
@@ -427,6 +472,11 @@ Type VolumeGroupSnapshotContentStatus struct {
         // +optional
         VolumeGroupSnapshotHandle *string
 
+        // VolumeSnapshotHandles is a list of CSI "snapshot_id" returned by the
+	// CSI driver to identify snapshots on the storage system.
+        // +optional
+        VolumeSnapshotHandles []string
```

Comment:
It should somehow capture which snapshot handle comes from which PV (or volume_handle). The controller needs to link them somehow and `PVVolumeSnapshotContentList` does not help here, because VolumeSnapshotContent does not exist yet.

Reply:
Ok. I changed this to VolumeSnapshotHandlePairList.  It is a list of volumeHandle/snapshotHandle pairs.